docs/en/dev/objectsschema.md

```diff
@@ -200,7 +200,12 @@ Attributes:
 * `common.read`   (boolean, mandatory) - true if state is readable
 * `common.write`  (boolean, mandatory) - true if state is writable
 * `common.role`   (string,  mandatory) - role of the state (used in user interfaces to indicate which widget to choose, see below)
-* `common.states` (optional) attribute of type number with the object of possible states `{'value': 'valueName', 'value2': 'valueName2', 0: 'OFF', 1: 'ON'}` or (supported up from admin5) an states array, like `['Start', 'Flight', 'Land']`
+* `common.states` (optional) - provide more context on allowed values for states with datatypes string and number:
+  * for numbers **without** provided common.min/common.max: contains the list of allowed number values and their textual mapping as object in form `{0: 'OFF', 1: 'ON', '-1': 'whatever'}`. Only these values should be allowed
+  * for numbers **with** provided common.min **and/or** common.max: the allowed number range is definedby min/max, this attribute contains a list of "special" number values and their textual mapping as object like `{0: 'OFF', 254: 'ON', 255: 'BLINK'}` (min=0, max=255).

```

Comment:
What's the range if only min or max are given?

Reply:
As now too ... 
* min <= x < "Infinity" or 
* -Infinity > x >= max

Reply:
DO we need to write that down or is this common sense?

Reply:
I added text